tools/build/Makefile

```diff
@@ -23,31 +23,13 @@ PKGGEN=$(abspath $(SRCDIR)/../rebuild.sh)
 PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/'}
 PYTHON_DIR=$(DSTDIR)/lib/python$(PY_VERSION)/site-packages/
 
-all: environment template enclavekey packages
+all: template enclavekey packages
```

Comment:
You could remove environment from .PHONY target

Reply:
Done